doc/api/querystring.md

```diff
@@ -119,7 +119,7 @@ given `obj` by iterating through the object's "own properties".
 
 It serializes the following types of values passed in `obj`:
 {string|number|boolean|string[]|number[]|boolean[]}
-Any other input values will be coerced to empty strings.
+Any other input values (including `null` and `undefined`) will be coerced to empty strings.
```

Comment:
If you run `make lint-md` (or `vcbuild lint-md` on Windows), this should report that the line is too long. Can you wrap it at 80 characters?